docs/pages/bare/updating-your-app.mdx

```diff
@@ -105,20 +106,20 @@ In **Expo.plist**, you'll need to add the following, replacing `your-channel-nam
 </dict>
 ```
 
-This configuration is also generated by `expo prebuild` if you set the `requestHeaders` property in your **app.json**: 
+This configuration is also generated by `expo prebuild` if you set the `requestHeaders` property in your **app.json**:
 
 ```json app.json
 {
   "expo": {
-  /* @hide ... your existing configuration */ /* @end */
+    /* @hide ... your existing configuration */ /* @end */
     "updates": {
       /* @hide ... */ /* @end */
       "requestHeaders": {
         "expo-channel-name": "your-channel-name"
-      }
-      /* @hide ... */ /* @end */
-    }
-  /* @hide ... */ /* @end */
+      } /* @end */
+      /* @hide ... */
+    } /* @end */
+    /* @hide ... */
```

Comment:
It was intentional to include `{` character in a hide block here? The idea behind original code was to display three dots, but do not include them in copy string.

Reply:
No, you are right, the intention wasn't to include `}` in the hide block. I've fixed this, thanks!